resources/qml/delegates/PlayableMediaMessage.qml

```diff
@@ -25,7 +25,7 @@ Item {
     required property string filesize
     property double divisor: isReply ? 4 : 2
     property int tempWidth: originalWidth < 1? 400: originalWidth
-    implicitWidth: type == MtxEvent.VideoMessage ? Math.round(tempWidth*Math.min((timelineView.height/divisor)/(tempWidth*proportionalHeight), 1)) : 500
+    implicitWidth: type == MtxEvent.VideoMessage ? Math.round(tempWidth*Math.min(((timelineView.height+Qt.inputMethod.keyboardRectange.height)/divisor)/(tempWidth*proportionalHeight), 1)) : 500
```

Comment:
typo, rectangle. You might want to make `timelineView.height+Qt.inputMethod.keyboardRectange.height` a property in the timelineView though. Like contentMaxSize or so.